content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,187 @@
+- :guilabel:`Display Time`: determine how many seconds a check-in/check-out confirmation message
+  remains on the kiosk screen before returning to the main check in screen.
+- :guilabel:`Attendance Kiosk Url`: Odoo generates a unique web address (URL) to use a device as a
+  kiosk, without having to sign in to the Odoo database. When setting up a kiosk device, navigate to
+  this unique web address in a web browser to present the *Attendances* app kiosk.
+
+  .. important::
+     These kiosk URLs are **not** secured with any type of access code. Anyone who has the URL can
+     access the *Attendances* app kiosk. If the URL is compromised for any reason, such as in the
+     event of a security breach, click :guilabel:`Generate a new Kiosk Mode URL`, located beneath
+     the link, to generate a new URL, and update the kiosk, accordingly.
+
+Kiosk mode
+==========
+
+Entering *iosk Mode* is **only** available for users with specific :ref:`access rights
+<attendances/access-rights>`.
+
+*Kiosk Mode* can be activated in three different ways:
+
+#. Navigate to the :menuselection:`Attendances app`, and click :guilabel:`Kiosk Mode` in the top
+   menu. The device then enters *Kiosk Mode*.
+#. Navigate to the :menuselection:`Attendances app --> Configuration`. In the :guilabel:`Kiosk
+   Settings` section, click the link in the :guilabel:`Attendance Kiosk Url` field. A new tab opens
+   in *Kiosk Mode*. It is recommended to close the database tab to keep the database secure.
+#. Navigate to the :menuselection:`Attendances app --> Configuration`. In the :guilabel:`Kiosk
+   Settings` section, click :guilabel:`Copy` next to the link beneath the :guilabel:`Attendance
+   Kiosk Url` field. Paste this URL into a web browser on the kiosk device, and navigate to it.
+
+.. image:: kiosks/kiosk-url.png
+   :align: center
+   :alt: The Attendances Kiosk URL field in the settings section of the Attendances application.
+
+As a security measure, once a device is in *Kiosk Mode*, it is not possible to exit *Kiosk Mode*,
+and go back into the database without signing back in.
+
+To exit *Kiosk Mode*, click the back button in the web browser. This logs the user out of the
+database, and returns to the main log in screen. This adds an extra layer of security, preventing
+anyone from accessing the database.
+
+.. _attendances/kiosk-mode-entry:
+
+Check in and out with a kiosk
+=============================
+
+Badge
+-----
+
+To check in or out using a badge, tap the :icon:`fa-camera` :guilabel:`Tap to scan` image in the
+center of the kiosk.
+
+.. image:: kiosks/scan-badge.png
+   :align: center
+   :alt: The Attendances Kiosk view displaying the scan badge image.
+
+Then, scan the barcode on the badge using the method configured in the :ref:`Kiosk Settings
+<attendances/kiosk-settings>` section of the configuration menu. Options are a dedicated barcode
+scanning device, or the kiosk's front or back camera.
```

Comment:
Redundant to list these options, as they are already covered in the referenced section
```suggestion
<attendances/kiosk-settings>` section of the configuration menu.
```